jekyll/_cci2/configuration-reference.md

```diff
@@ -31,7 +31,7 @@ Key | Required | Type | Description
 docker | Y <sup>(1)</sup> | List | Options for [docker executor](#docker-executor)
 machine | Y <sup>(1)</sup> | Map | Options for [machine executor](#machine-executor)
 steps | Y | List | A list of [steps](#steps) to be performed
-working_directory | Y | String | What directory to run the steps in (default: depends on executor)
+working_directory | Y | String | What directory to run the steps in. Can be short-handed as `workDir`.
```

Comment:
The issue is that people are confused by snippets in the wild.  The thinking here is it's more important to prevent confusion then worry about some nebulous future where we may deprecate this token.